<launch>

    <node pkg="yasht_safety" name="safety_node" type="safety_node" output="screen"/>

    <rosparam command="load" file="$(find f110_reactive_methods)/config/config.yaml" />

    <env name="ROSCONSOLE_CONFIG_FILE" value="rosconsole_config.conf"/>

    <node pkg="f110_reactive_methods" name="follow_the_gap_node" type="follow_the_gap_node" output="screen"/>

</launch>